implementation/src/main/java/com/teampc/utils/FXUtils.java

```diff
@@ -0,0 +1,30 @@
+package com.teampc.utils;
+
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+
+import java.io.IOException;
+import java.util.function.Consumer;
+
+/**
+ * Created by adufrene on 11/9/15.
+ */
+public class FXUtils {
+
+   public static Stage switchToScreen(Stage stage, String layoutFilename) throws IOException {
```

Comment:
@adufrene can you add a javadoc so we know when this function is supposed to be used?